### DISCUSSION TITLE
Better Calendar

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#387] [Ideas]
Hi Team,

I am trying to create a customer and it asks for a birthdate. Now there are two things

1) There is no way one can ignore the birthdate as the most customer never wants to give that. 
2) Some customers will give only Day and Month of their birthday to avoid letting others know their age.
3) Please have a better calendar option as I want to click 100+ times to go to the year 1980. There is a better JS calendar, which allows choosing our own month, own year and then dates. 
4) Birthdate doesn't require time.

**Comment by Blair2004** [comment]
This has been recently fixed. 
Thank you.
![image](https://user-images.githubusercontent.com/5265663/147220951-ff0d554f-a7e9-465c-a1c0-4d6bd2ad0c28.png)
